doc/GuideForCommitters.md

```diff
@@ -21,13 +21,41 @@ SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0-only WITH Classpath-ex
 
 # Eclipse OMR Committer Guide
 
-This document outlines some general rules for Eclipse OMR committers to
-follow when reviewing and merging pull requests and issues.  It also provides
-a checklist of items that committers must ensure have been completed prior to
-merging a pull request.
+This document provides information useful for Eclipse OMR committers.
 
+It outlines minimum participation requirements for committers, general rules
+for committers to follow when reviewing and merging pull requests and issues,
+and provides a checklist of items that committers must ensure are completed
+prior to merging a pull request.
 
-## General Guidelines
+## Minimum Committer Participation Standard
+
+Eclipse OMR committers are expected to meet at least **one** of the following
+criteria within a consecutive 9-month period in order to maintain their active
+committer status:
+
+* Review and merge at least one pull request, or
+* Author and contribute at least one pull request, or
+* Participate in at least 25% of
+[OMR Architecture Meetings](https://github.com/eclipse/omr/issues/2316), or
+* Other documented contributions not reflected in the above may be considered
+on a case-by-case basis by Eclipse OMR project leadership.  For example, significant
+contributions to the design of a feature, or assistance resolving a defect,
+or project advocacy work may apply.
+
+Committers that are unable to meet the minimum participation standard will be
```

Comment:
let's not make this about ability: "Committers that do not meet the minimum participation standards can be retired..."

Reply:
Fixed